File: 009-斐波那契数列/009-rectCover.py
# -*- coding:utf-8 -*-
class Solution:
    def rectCover(self, number):
        # write code here
        if number<=2:
            return number
        a,b=1,2
        while number>=3:
            a,b=b,a+b
            number-=1
        return b

File: 009-斐波那契数列/009-rectCover.cpp
class Solution {
public:
    int rectCover(int number) {
        if (number<=2){
            return number;
        }
        int a=1;
        int b=2;
        while(number>=3){
            int temp=a;
            a=b;
            b=temp+b;
            number--;
        }
        return b;
    }
};
